docs/manual/scala/guide/advanced/code/build-update-akka.sbt

```diff
@@ -0,0 +1,38 @@
+// #akka-update
+// The newer Akka version you want to use.
+val akkaVersion = "2.6.0-RC2"
+
+// Akka dependencies used by Lagom
+libraryDependencies ++= Seq(
+  "com.typesafe.akka" %% "akka-actor"                  % akkaVersion,
+  "com.typesafe.akka" %% "akka-actor-testkit-typed"    % akkaVersion,
+  "com.typesafe.akka" %% "akka-actor-typed"            % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster"                % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster-sharding"       % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster-sharding-typed" % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster-tools"          % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster-typed"          % akkaVersion,
+  "com.typesafe.akka" %% "akka-coordination"           % akkaVersion,
+  "com.typesafe.akka" %% "akka-discovery"              % akkaVersion,
+  "com.typesafe.akka" %% "akka-distributed-data"       % akkaVersion,
+  "com.typesafe.akka" %% "akka-multi-node-testkit"     % akkaVersion,
+  "com.typesafe.akka" %% "akka-persistence"            % akkaVersion,
+  "com.typesafe.akka" %% "akka-persistence-query"      % akkaVersion,
+  "com.typesafe.akka" %% "akka-persistence-typed"      % akkaVersion,
+  "com.typesafe.akka" %% "akka-protobuf"               % akkaVersion,
+  "com.typesafe.akka" %% "akka-protobuf-v3"            % akkaVersion,
```

Comment:
I think we managed to drop `akka-protobuf` so it should be removed from here.

It's not great to have to maintain this list by hand. Any idea on how to automate it?

Reply:
I mean, I'd rather give users a way to list the dependencies under `com.typesafe.akka` than have them add all the dependencies (even when they don't need them).

Reply:
Ok, I just noticed the sentence "Of course, other Akka artifacts can be added transitively. Use sbt-dependency-graph..." in the docs. 

In that case, I'd keep this list a lot smaller and with a comment:

```
// #akka-update
// The newer Akka version you want to use.
val akkaVersion = "2.6.0-RC2"

// Akka dependencies used by Lagom
libraryDependencies ++= Seq(
  "com.typesafe.akka" %% "akka-actor"                  % akkaVersion,
  "com.typesafe.akka" %% "akka-actor-testkit-typed"    % akkaVersion,
  "com.typesafe.akka" %% "akka-actor-typed"            % akkaVersion,
  "com.typesafe.akka" %% "akka-cluster"                % akkaVersion,
  "com.typesafe.akka" %% "akka-stream-testkit"         % akkaVersion % Test,
  "com.typesafe.akka" %% "akka-testkit"                % akkaVersion % Test,
  // Use "sbt-dependency-graph" or any other dependency report generator to 
  // make sure you add all the necessary dependencies on this list.
)
```

Reply:
> I think we managed to drop `akka-protobuf` so it should be removed from here.

I think I was tricked by the whitelist validation. `akka-protobuf` is stilll there:

https://github.com/lagom/lagom/blob/709e6663f05f1212fad18b7b3da2a87194702a7b/project/Dependencies.scala#L385

I could not use my own recommendation to use `sbt-dependency-graph` because it is not working with sbt 1.3. 😢 

Anyway, I think the correct list will be:

```scala
"com.typesafe.akka" %% "akka-actor"                  % Versions.Akka
"com.typesafe.akka" %% "akka-remote"                 % Versions.Akka
"com.typesafe.akka" %% "akka-cluster"                % Versions.Akka
"com.typesafe.akka" %% "akka-cluster-sharding"       % Versions.Akka
"com.typesafe.akka" %% "akka-cluster-tools"          % Versions.Akka
"com.typesafe.akka" %% "akka-distributed-data"       % Versions.Akka
"com.typesafe.akka" %% "akka-serialization-jackson"  % Versions.Akka
"com.typesafe.akka" %% "akka-multi-node-testkit"     % Versions.Akka
"com.typesafe.akka" %% "akka-persistence"            % Versions.Akka
"com.typesafe.akka" %% "akka-persistence-query"      % Versions.Akka
"com.typesafe.akka" %% "akka-slf4j"                  % Versions.Akka
"com.typesafe.akka" %% "akka-stream"                 % Versions.Akka
"com.typesafe.akka" %% "akka-protobuf-v3"            % Versions.Akka
"com.typesafe.akka" %% "akka-actor-typed"            % Versions.Akka
"com.typesafe.akka" %% "akka-cluster-typed"          % Versions.Akka
"com.typesafe.akka" %% "akka-persistence-typed"      % Versions.Akka
"com.typesafe.akka" %% "akka-cluster-sharding-typed" % Versions.Akka
"com.typesafe.akka" %% "akka-testkit"                % Versions.Akka
"com.typesafe.akka" %% "akka-stream-testkit"         % Versions.Akka
"com.typesafe.akka" %% "akka-actor-testkit-typed"    % Versions.Akka
"com.typesafe.akka" %% "akka-discovery"              % Versions.Akka
```